src/components/HTTPHeader.jsx

```diff
@@ -64,8 +64,9 @@ export default function HTTPHeader({ type, path }) {
       setTimeout(() => setCopy(false), 3000);
     }
   }, [copied]);
-  const BASE_URL = "https://top.gg/api/";
-  const fullUrl = new URL(path, BASE_URL).href;
+  const BASE_URL = "https://top.gg/";
+  path = `${path.startsWith("/") ? "" : "/"}${path}`
+  const fullUrl = new URL(`api${path}`, BASE_URL).href;
```

Comment:
This prevents us from ever being able to pass absolute urls into this `HTTPHeader` component which is one of the requirements I want to be able to keep for now

Reply:
How does this look?